extensions/wikia/EditPageLayout/templates/EditPageLayout_CodeButtons.php

```diff
@@ -1,20 +1,7 @@
-<?php
-$dropdown = array(array(
-	"id" => "wpDiff",
-	"accesskey" => wfMessage('accesskey-diff')->escaped(),
-	"text" => wfMessage('showdiff')->escaped()
-));
-?>
-<?= F::app()->renderView('MenuButton',
-	'Index',
-	array(
-		'action' => array(
-			'text' => wfMessage('savearticle')->escaped(),
-			'class' => 'codepage-publish-button',
-			'id' => 'wpSave',
-		),
-		'name' => 'submit',
-		'class' => 'primary',
-		'dropdown' => $dropdown
-	)
-) ?>
+<?php if ( !empty( $approveCheckbox ) ): ?>
+	<label for="wpApprove" class="approve-changes-label">
+		<input id='wpApprove' name='wpApprove' type='checkbox' value='1'/>
+		<?= wfMessage( 'content-review-edit-page-checkbox-label' )->escaped() ?>
```

Comment:
would be nice to be consistent within file whether to add semicolons at the end of lines or not :)